tests/math/test_functions.py

```diff
@@ -137,8 +137,9 @@ def test_return_numpy_box(self):
     def test_get_deep_interface(self):
         """Test get_deep_interface returns the interface of deep values."""
         assert fn.get_deep_interface([()]) == "builtins"
-        assert fn.get_deep_interface(([1, 2], [3, 4])) == "builtins"
+        assert fn.get_deep_interface(([1, 2], [3, 4])) == "numpy"

```

Comment:
For context, qml.math.get_interface([1, 2]) gives you `numpy` instead of `builtins`. IMO the behavior of `get_deep_interface` should be consistent with `get_interface` so that's why I'm editing the test case here.

Reply:
it is a little strange that `get_deep_interface` has inconsistent behaviour with `get_interface` though; notably when you do `get_interface(())` you will also get `numpy` instead of `builtins`. Should we make the behaviour consistent?

Reply:
I think as long as it isn't breaking anything we want to support, we are fine.